### ISSUE TITLE
Drop IDL merged into CSSOM

### ISSUE THREAD

**Issue by tidoust** [WICG/construct-stylesheets#139]
Most of the IDL defined in the Constructable Stylesheet Objects has now been merged into CSSOM (done in https://github.com/w3c/csswg-drafts/commit/0aca629bc2ff93d364eb363fa751c065187153fe)

It would be good to drop the IDL so as not to create duplicate IDL definitions.

I suspect that the plan is to merge the remaining IDL that defines the `adoptedStyleSheets` property into the DOM spec, and to drop this spec afterwards. It's totally fine to leave this issue open and unaddressed in the meantime. I'm raising this issue for tracking purpose to be alerted when that has been done (so that we can drop the corresponding IDL patch in Webref)

**Comment by tidoust** [comment]
With https://github.com/w3c/csswg-drafts/commit/3664d57f7f2265f82f99c2c0cfb56a344c5431d0, the remaining IDL has now been fully integrated into CSSOM, which I understand concludes work on this spec. I'll drop it from browser-specs as a result (see https://github.com/w3c/browser-specs/pull/353). Let me know if there are plans to continue work on this specification!

**Comment by domenic** [comment]
Yeah, I think it would be good to convert this spec into a redirect stub, and probably do a pass through the open issues list to see if anything should be ported over to csswg-drafts. Then the repository could be archived. /cc @mfreed7
